CONTRIBUTING.md

```diff
@@ -92,7 +92,20 @@ To develop locally:
    - Use `openssl rand -base64 32` to generate a key and add it under `NEXTAUTH_SECRET` in the `.env` file.
    - Use `openssl rand -base64 24` to generate a key and add it under `CALENDSO_ENCRYPTION_KEY` in the `.env` file.
 
-6. Start developing and watch for code changes:
+6. Setup Node
+   If your Node version does not meet project's requirements as instructed by the docs, "nvm" (Node Version Manager) allows to use Node at the version required by the project:
```

Comment:
"the project's requirements"
"allows using Node"

Reply:
🫡 Yessir, resolved in [refactor: correct use of English](https://github.com/calcom/cal.com/pull/11551/commits/c5ae58af133f8c2b84d7f89563af47867dbee82f)